### PR TITLE
[close-loop] bug-2180549, test multisite mixed-mode upgrade

### PR DESCRIPTION
**Pull request by viduship**
bug- https://bugzilla.redhat.com/show_bug.cgi?id=2180549

Testcase: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574647

Passed logs : 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-J7VELE/

### CODE REVIEW

**Review by anrao19 — DISMISSED** [review]

**Review by ckulal — COMMENTED** [review]
@viduship , any reason why we are making changes to existing suite,
not creating new suite for mixed mode upgrade

**Comment by viduship** [comment]
> @viduship , any reason why we are making changes to existing suite, not creating new suite for mixed mode upgrade

@ckulal , 
- Adding a new suite might add more time either in weekly or regression.
- Also, this way of uploading is similar to how a customer performs an upgrade, that is one site is upgraded, performs the IOs, and then upgrades the next site.

File: suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml [hunk 1]
```diff
@@ -363,16 +363,11 @@ tests:
 
   - test:
       abort-on-fail: true
-      desc: Multisite upgrade
+      desc: rgw-MS mixed-mode ceph upgrade
```
Comment (mkasturi18) [inline suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml:366]: @viduship , could you add details on the site being upgraded in both places, L366 & L422

**Review by mkasturi18 — CHANGES_REQUESTED** [review]

File: suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml (hunk 1 above)
Comment (viduship) [reply suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml]: done @mkasturi18

**Review by hmaheswa — APPROVED** [review]

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**